connector/protobuf/src/main/scala/org/apache/spark/sql/protobuf/utils/ProtobufOptions.scala

```diff
@@ -168,6 +168,28 @@ private[sql] class ProtobufOptions(
   // instead of string, so use caution if changing existing parsing logic.
   val enumsAsInts: Boolean =
     parameters.getOrElse("enums.as.ints", false.toString).toBoolean
+
+  // Whether to unwrap the struct representation for well known primitve wrapper types when
+  // deserializing. By default, the wrapper types for primitives (i.e. google.protobuf.Int32Value,
+  // google.protobuf.Int64Value, etc.) will get deserialized as structs. We allow the option to
+  // deserialize them as their respective primitives.
+  // https://protobuf.dev/reference/protobuf/google.protobuf/
+  //
+  // For example, given a message like:
+  // ```
+  // syntax = "proto3";
+  // message = Example {
+  //   google.protobuf.Int32Value val = 1;
+  // }
+  // ```
+  //
+  // The message Example(Int32Value(1)) would be deserialized by default as
+  // {val: {value: 1}}
+  //
+  // However, with this option set, it would be deserialized as
+  // {val: 1}
+  val unwrapWellKnownTypes: Boolean =
+    parameters.getOrElse("unwrap.protobuf.wkt", false.toString).toBoolean
```

Comment:
Lets find a better name for this. 
`unwrap.primitive.value.types` or something on those lines? Better to avoid short forms like `wkt`.

Reply:
yes, definitely. i updated it, let me know what you think